docusaurus/docs/develop/path/4_path_localnet.md

```diff
@@ -0,0 +1,347 @@
+---
```

Comment:
Couldn't get it to work :(

<img width="1766" height="1232" alt="Screenshot 2025-08-28 at 8 54 03 PM" src="https://github.com/user-attachments/assets/1d20bf02-14f4-4b95-a90c-b77feed60028" />